src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.json

```diff
@@ -1,12 +1,27 @@
 {
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
   "dependencies": {
+    "NuProj": "0.10.4-beta-gf7fc34e7d8",
     "Microsoft.Build.Framework": "0.1.0-preview-00005",
     "Microsoft.Build.Tasks.Core": "0.1.0-preview-00005",
     "Microsoft.Build.Utilities.Core": "0.1.0-preview-00005",
+    "Microsoft.CSharp": "4.0.1-beta-23328",
+    "System.Linq": "4.0.1-beta-23328",
+    "System.Xml.XDocument": "4.0.11-beta-23328",
+    "System.IO.Compression": "4.0.1-beta-23328",
+    "System.IO.Compression.ZipFile": "4.0.1-beta-23328",
+    "System.Xml.ReaderWriter": "4.0.11-beta-23328",
+    "System.Xml.XPath.XmlDocument": "4.0.0",
+    "System.Reflection": "4.1.0-beta-23328",
+    "System.Reflection.Metadata": "1.1.0",
```

Comment:
Most of these are redundant to `Microsoft.NETCore.Console`.  In some cases they may contradict what is in there (specifically looking at Reflection.Metadata here).  Can you review this to make sure it is the minimal set of references?